packages/azure-openapi-validator/autorest/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft.azure/openapi-validator",
-  "version": "2.1.5",
+  "version": "2.1.6",
```

Comment:
This shouldn't be changed in this PR: the only changes are in `openapi-validator-rulesets` dir.

Reply:
In general, observe:

![image](https://github.com/Azure/azure-openapi-validator/assets/4429827/bbfa391c-e579-4e57-a50c-24cc621728d5)

Observe that each of these directories has `packages.json` and they have names:

![image](https://github.com/Azure/azure-openapi-validator/assets/4429827/b541cbba-0e6f-4647-bdcf-e3f6541bbfbf)

These are the packages, so the directories are boundaries that denote which needs an update.

You can also find the published package versions on npm and see their contents:
https://www.npmjs.com/package/@microsoft.azure/openapi-validator-rulesets

Can you update the contributing guide we have to make it clearer?

Reply:
Yup lets improve the doc so even first timers can cruise thru this.